.gitlab-ci.yml

```diff
@@ -48,7 +48,7 @@ variables:
   CARGO_INCREMENTAL: 0
   DOCKER_OS: "debian:bullseye"
   ARCH: "x86_64"
-  CI_IMAGE: "paritytech/ci-linux:production"
+  CI_IMAGE: "paritytech/ci-linux@sha256:c725397b3edef45ad8dfa6a829b35e7c908b74ed532929bb6ad24cca1129dc8e" # ci-linux:1.70.0-bullseye 2023-06-19
```

Comment:
DQ but why do you not tag the `paritytech/ci-linux` image itself like that?

Reply:
The `ci-linux:1.70.0-bullseye` tag does exist, and we do intend on using instead of `ci-linux:production` in the future! There's just a whole heap of infrastructure changes we need to work through first before that's feasible; I'm hoping we can get there soon.

Reply:
So for now we're still using the old process of temporarily pinning the image hash during upgrades and then reverting to `ci-linux:production` once everything has worked out.